src/Nox.Generator/Application/Queries/Query.template.cs

```diff
@@ -21,6 +21,7 @@ public class Get{{entity.PluralName}}QueryHandler : IRequestHandler<Get{{entity.
 
     public Task<IQueryable<O{{entity.Name}}>> Handle(Get{{entity.PluralName}}Query request, CancellationToken cancellationToken)
     {
-        return Task.FromResult((IQueryable<O{{entity.Name}}>)DataDbContext.{{entity.PluralName}});
+        var item = (IQueryable<O{{entity.Name}}>)DataDbContext.{{entity.PluralName}}{{if isVersioned}}.Where(r => !(r.Deleted == true)){{end}};
```

Comment:
do we need r.Deleted == true? r.Deleted should be enough?

Reply:
r.Deleted won't work as Deleted is a nullable field and it cannot automatically convert bool? to bool.